### ISSUE TITLE
getting error while trying to get headers

### ISSUE THREAD

**Issue by K1NGPIN97** [sonya75/starbucks-botdetection-cracked#1]
File "C:\Users\Administrator\Desktop\starbucks\sensordata.py", line 95, in processrawscript
    m=re.search(" H=(.*?\]);",x).group(1)
AttributeError: 'NoneType' object has no attribute 'group'

**Comment by sonya75** [comment]
Yes, I am aware of the issue. But I have been very busy this week. I will try to post an update this weekend or early Monday.

**Comment by GunnGrave** [comment]
i am having the same error

**Comment by sonya75** [comment]
Hi, sorry for the delay. Been very busy. Got some free time for a few days now. I will post an update today or tomorrow.

**Comment by GunnGrave** [comment]
any news about the regex

**Comment by ostapvolanuk** [comment]
> Hi, sorry for the delay. Been very busy. Got some free time for a few days now. I will post an update today or tomorrow.

Hey! Do you have a static function file your script is working with?

**Comment by Yiming828** [comment]
I have same issue

**Comment by Petr2282** [comment]
> Yes, I am aware of the issue. But I have been very busy this week. I will try to post an update this weekend or early Monday.

Hi man do you have telegram need to discuss?
